### PR TITLE
Added Jackknives, Plots, and Stats

### PR DESCRIPTION
**Pull request by duncanjmr**
Added setup, version, LISENCE and .travis.yml.
Added Modules/Files:
1. jackknife.py: Module for jackknifing data and saving results.
2. jkf_stats.py: Module for loading and doing statistical tests on jackknife files.
3. plots.py: Module for jackknife data visualization.
4. utils.py: Extra functions + class for storing jackknife data efficiently.
Also added tests and (underdeveloped) example notebooks.

### CODE REVIEW

File: hera_stats/jkf_stats.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
+# Statistics and such for hera data
+# Duncan Rocha, June 2018
+
+import numpy as np
+import pickle as pkl
+from scipy import stats as spstats
+import utils
+
+class jkf_stats():
```
Comment (nkern) [inline hera_stats/jkf_stats.py]: class names should be capitalized: `JKF_Stats`

File: hera_stats/plots.py [hunk 2]
```diff
@@ -0,0 +1,599 @@
+# Plots for visualizing data
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pickle as pkl
+from jkf_stats import jkf_stats
+import utils
+
+class plots():
```
Comment (nkern) [inline hera_stats/plots.py]: capitalization: `Plots`

**Comment by duncanjmr** [comment]
Please work... :(

**Comment by duncanjmr** [comment]
Finally!!!

**Pull request closed without merging**